### PR TITLE
feat: add utm content to share link

### PR DESCRIPTION
**Pull request by christian14b**
This feature adds utm_content to the share buttons in Thanks Page.

### CODE REVIEW

File: src/components/Checkout/SocialShare.vue [hunk 1]
```diff
@@ -200,13 +204,19 @@ export default {
 		shareMessage() {
 			return this.message.trim() || this.suggestedMessage;
 		},
+		utmContent() {
+			if (this.isGuest) return 'guest';
+			if (this.lender?.public) return this.lender?.inviterName;
```
Comment (dyersituations) [inline src/components/Checkout/SocialShare.vue:209]: Do we need to also check if `this.lender?.inviterName` is defined here?

Comment (emuvente) [reply src/components/Checkout/SocialShare.vue:209]: We could check that. That value is always defined in the database, but there's a chance that an error could occur while trying to resolve it.

**Review by emuvente — APPROVED** [review]
👍 Looks good!

**Review by emuvente — COMMENTED** [review]
@christian14b could you also please add these changes to SocialShareV2? That's going to become the new default.

**Comment by christian14b** [comment]
Will do!

**Comment by christian14b** [comment]
@emuvente changes are now in `SocialSharev2` component too

**Review by emuvente — APPROVED** [review]
Looks great! Thanks for the additional changes!

**Review by edwinf-barreram — APPROVED** [review]
LGTM 👍

File: src/components/Checkout/SocialShare.vue [hunk 2]
```diff
@@ -226,13 +236,13 @@ export default {
...
 				title: `A loan for ${this.selectedLoan.name}`,
-				url: `${this.shareLink}?utm_source=linkedin.com&utm_medium=social&utm_campaign=social_share_checkout`
+				url: `${this.shareLink}&utm_source=linkedin.com&utm_medium=social&utm_campaign=social_share_checkout`
 			});
 		},
 		twitterShareUrl() {
 			return this.getFullUrl('https://twitter.com/intent/tweet', {
 				text: this.shareMessage,
-				url: `${this.shareLink}?utm_source=t.co&utm_medium=social&utm_campaign=social_share_checkout`,
+				url: `${this.shareLink}&utm_source=t.co&utm_medium=social&utm_campaign=social_share_checkout`,
```
Comment (emuvente) [inline src/components/Checkout/SocialShare.vue:245]: This change is also needed in the `copyLink` method

**Review by emuvente — COMMENTED** [review]
@christian14b Could you please resolve the merge conflicts? Once that's done this can be merged!

File: src/components/Checkout/SocialShare.vue (hunk 2 above)
Comment (christian14b) [reply src/components/Checkout/SocialShare.vue:245]: Sure! working on it

**Review by emuvente — APPROVED** [review]
👍 Thanks!

**Pull request merged**